.mypy.ini

```diff
@@ -0,0 +1,10 @@
+[mypy]
+python_version = 3.8
+exclude = .*plugins\/.*
+
+; The following is REQUIRED - `exclude` DOES NOT do what you think it does. It
+; ONLY ignores files encountered while recursively discovering files to check.
+; Without this option, mypy will ALWAYS include 'atomic_reactor/plugins/post_rpmqa.py'
+; Yes it did take me ~1.5 hrs to figure this out.
```

Comment:
Hmm, so what makes atomic_reactor/plugins/post_rpmqa.py special? Does the exclusion work for all plugins except that one?

Reply:
no idea, but I hit the same

```
mypy atomic_reactor/ --install-types  --ignore-missing-imports --exclude "plugins/" 
...
atomic_reactor/plugins/post_rpmqa.py:24: error: Incompatible types in assignment (expression has type "str", base class "Plugin" defined the type as "None")
```

Reply:
> Does the exclusion work for all plugins except that one?

Yes. Based on how I'm *actually* able to ignore it, I'd say it's imported in another module somewhere.

Reply:
Ack, that seems plausible :)